README.md

```diff
@@ -0,0 +1,209 @@
+# Proposing and discussing changes
+
+## Introduction
+
+The repository first objective is to give a place where community memebers can document their design ideas for WoT projects. 
+The structure of this repository is heavily inspired by the [golang proposal system](https://github.com/golang/proposal).
+
+Each design change must be discussed on a proposal issue. When needed, a document expressing the change motivations is merged inside this repository and refrenced to the issue with a precise [naming schema](#design-documents).
+
+Users must adhere to the [code of conduct](#code-of-conduct) of the community when discussing a proposal. Partecipating to the project evolutions needs to be a pleasant experience for everyone. We pursue a friendly, welcoming environment where everyone can express their ideas, always feeling at home. 
+
+This document will describe the whole proposal workflow, from the opening of an issue, to the acceptance or refusal of design changes through discussion and engagement of the community. 
+
+## What should be considered as a 'Proposal' ?
+
+Proposals are essentially design changes of any of the wot-open-source-project repositories.
+
+Bugs, chores and changes related to external components are not considered as such. These issues must be tagged accordingly on their corresponding repositories.
+
+If you doubt the issue nature, and you feel it could be considered a design change to one project, you can always file a proposal.
```

Comment:
Don't understand what you mean by this.

Reply:
I would like to emphasize that the issue posted here are only related to proposals, and not bugs.
If someone is not able to understand if the issue is a proposal or not (maybe a modification that is more related to a bug rather than a feature), they should be able to still post it. Maintainers would remove it if it is not considerable as a proposal.

Do you think we should remove this part ?